app/assets/stylesheets/components.scss

```diff
@@ -554,6 +554,12 @@ a.status__content__spoiler-link {
   .status__display-name strong {
     color: lighten($color1, 26%);
   }
+
+  .status__avatar-boost {
+    display: inline-block;
+    vertical-align: middle;
+    margin-right: 2px;
```

Comment:
The margin in the screenshot looks a little small for me. Maybe change this to something more like `5px`?